npm/pkg/dataplane/policies/policymanager_windows.go

```diff
@@ -357,6 +358,16 @@ func getSettingsFromACL(policy *NPMNetworkPolicy) ([]*NPMACLPolSettings, error)
 		}
 		hnsRules[i] = rule
 	}
+
+	// fixes #1881
+	hnsRules[len(policy.ACLs)] = &NPMACLPolSettings{
+		Id:        policy.ACLPolicyID,
+		Action:    hcn.ActionTypeAllow,
```

Comment:
You are allowing all rules, you should add the source ip as node ip here right ? 
Add a comment on this too .